src/main/groovy/com/avioconsulting/mule/linter/rule/configuration/ExcessiveLoggersRule.groovy

```diff
@@ -0,0 +1,66 @@
+package com.avioconsulting.mule.linter.rule.configuration
+
+import com.avioconsulting.mule.linter.model.Application
+import com.avioconsulting.mule.linter.model.configuration.FlowComponent
+import com.avioconsulting.mule.linter.model.rule.Rule
+import com.avioconsulting.mule.linter.model.rule.RuleViolation
+
+class ExcessiveLoggersRule extends Rule {
+    static final String RULE_ID = 'EXCESSIVE_LOGGERS_RULE'
+    static final String RULE_NAME = 'Excessive use of Sequential Loggers'
+    static final String RULE_VIOLATION_MESSAGE = 'Too many sequential loggers of same level in flow '
+
+    Map<String, Integer> excessiveLoggers = ["TRACE": 2, "DEBUG": 2, "INFO": 2, "WARN": 2, "ERROR": 2]
```

Comment:
We should move these strings to either be an enum or static constants, probably not on this class but maybe on the LoggerComponent model class?

Reply:
We'll try an Enum for now.